sdk/python/feast/on_demand_feature_view.py

```diff
@@ -68,8 +68,8 @@ class OnDemandFeatureView(BaseFeatureView):
     features: List[Field]
     source_feature_view_projections: Dict[str, FeatureViewProjection]
     source_request_sources: Dict[str, RequestSource]
-    transformation: Union[OnDemandPandasTransformation]
-    feature_transformation: Union[OnDemandPandasTransformation]
+    transformation: Union[PandasTransformation]
+    feature_transformation: Union[PandasTransformation]
```

Comment:
Sorry for repeating myself, just making sure... Do you intend to remove either `transformation` or `feature_transformation` parameters in another PR?

Reply:
oh yes let me remove them

Reply:
Can you change the type to `Union[PandasTransformation, SubstraitTransfromation]`? I think I missed it in my original PR.

Reply:
Done